File: BE/src/main/java/com/ssafy/e106/domain/subscriptionusage/dto/response/SubscriptionUsageReportSummaryResponse.java
package com.ssafy.e106.domain.subscriptionusage.dto.response;

public record SubscriptionUsageReportSummaryResponse(
    Integer windowDays,
    Integer totalUsedMinutes,
    Integer activeSubscriptionCount,
    Integer lowUsageSubscriptionCount,
    String mostUsedSubscriptionName,
    Integer mostUsedSubscriptionMinutes) {
}

File: FE/app/src/main/java/com/ssafy/e106/data/dto/subscriptionusage/SubscriptionUsageReportSummaryResponse.kt
package com.ssafy.e106.data.dto.subscriptionusage

import kotlinx.serialization.Serializable

@Serializable
data class SubscriptionUsageReportSummaryResponse(
    val windowDays: Int,
    val totalUsedMinutes: Int,
    val activeSubscriptionCount: Int,
    val lowUsageSubscriptionCount: Int,
    val mostUsedSubscriptionName: String? = null,
    val mostUsedSubscriptionMinutes: Int? = null,
)
